colossalai/amp/naive_amp/naive_amp.py

```diff
@@ -40,7 +43,7 @@ def step(self):
         return self.optim.step()
 
     def clip_grad_norm(self, model: nn.Module, max_norm: float):
-        pass
+        raise RuntimeError("Misuse error: you should specify the clip_grad_norm in the amp config")
```

Comment:
Perhaps it is clearer to say "NaiveAMP optimizer has executed gradient clipping during optimizer.step() if you have supplied `clip_grad_norm` in the amp_config, executing the method clip_grad_norm is not allowed."

Reply:
@FrankLeeeee done.